index.html

```diff
@@ -249,6 +250,24 @@ <h2>Dictionary <code><a>RTCRtpEncodingParameters</a></code> Members</h2>
         <code>ptime</code> MUST NOT be set; otherwise,
         <code>InvalidModificationError</code> MUST be thrown.</p>
       </dd>
+      <dt><dfn data-idl><code>maxFramerate</code></dfn> of type <span class=
+      "idlMemberType">double</span></dt>
+      <dd>
+        <p>When present, indicates the maximum frame rate that can be used to
+        send this encoding, in frames per second. The user agent is free to
+        allocate bandwidth between the encodings, as long as the
+        <code>maxFramerate</code> value is not exceeded.</p>
+        <p>If changed with <code>setParameters</code>, the new frame rate takes
+        effect after the current picture is completed; setting the max framerate
+        to zero thus has the effect of freezing the video on the next frame.</p>
+        <p>Upon setting with <code>addTransceiver</code> or
+        <code>setParameters</code>, if the value is less than 0.0,
+        <a>throw</a> a <code>RangeError</code>.</p>
```

Comment:
Maybe break this up into two:
1. Upon setting with addTransceiver, throw
2. Upon setting with setParameters, reject

Reply:
Done.